DataFormats/TrackSoA/src/classes_def.xml

```diff
@@ -5,9 +5,7 @@
   <class name="reco::TrackSoAView<pixelTopology::Phase1>"/>
 
   <class name="PortableHostCollection<reco::TrackLayout<pixelTopology::Phase1>>"/>
-  <class name="pixelTrack::TracksHostPhase1" ClassVersion="3">
-    <version ClassVersion="3" checksum="794224446"/>
-  </class>
+  <class name="pixelTrack::TracksHostPhase1"/>
```

Comment:
Why do you remove the class versions ?

Reply:
Because I asked in https://github.com/cms-sw/cmssw/pull/43964#discussion_r1491530401 after realizing all of these are class template instantiations (and thus I likely gave false advice in the earlier PR). For class templates the class version should be recored via `CMS_CLASS_VERSION()` macro in the class template definition.

Given that for now (like data taking of this year) to my understanding these classes are not intended for long term storage (e.g. AOD onwards), I'd suggest we investigate the various schema evolution aspects of the SoA and PortableCollection later.

Reply:
Ah... thanks for the clarification.

Should we remove the class version from the other `PortableCollection` as well ?
Or keep them there ?

Reply:
Removing from all `PortableCollection`s would be most consistent. The `ZVertexHost` is already dealt in https://github.com/cms-sw/cmssw/pull/43952, and IIUC this PR would deal with the other types used in the pixel reconstructions. I didn't check (and don't remember) if the ECAL or PF PRs added the class version for `PortableCollection` instantiations.

Reply:
I am going to change these types: `SiPixelDigiSoA`, `SiPixelClusterSoA`, `TrackingRecHitSoA` and `TrackSoA`. Also, I assume that `BeamSpotPOD` [here](https://github.com/borzari/cmssw/blob/AlpakavsCUDA/DataFormats/BeamSpot/src/classes_def.xml#L8-L10) should have the version removed as well. Can you confirm, please?

Reply:
> Also, I assume that `BeamSpotPOD` [here](https://github.com/borzari/cmssw/blob/AlpakavsCUDA/DataFormats/BeamSpot/src/classes_def.xml#L8-L10) should have the version removed as well. Can you confirm, please?

The class version of `BeamSpotPOD` should be kept, as that is a class by itself rather than a class template instantiation.